apps/mobile/src/libs/components/button/button.tsx

```diff
@@ -1,19 +1,102 @@
 import React from "react";
-import { Pressable } from "react-native";
+import { Pressable, View } from "react-native";
+import LinearGradient from "react-native-linear-gradient";
 
 import { Text } from "~/libs/components/components";
+import { BaseColor, GradientColor } from "~/libs/enums/enums";
 
+import { GradientText } from "../gradient-text/gradient-text";
 import { styles } from "./styles";
 
 type Properties = {
+	appearance?: "filled" | "outlined";
+	borderRadius: number;
```

Comment:
Let's remove this from props and set it directly. Don't think that we'll need to change it.